conversation_reconstruction_dataflow/ingest_revisions/dataflow_main.py

```diff
@@ -68,19 +71,36 @@ def run(known_args, pipeline_args):
   with beam.Pipeline(options=pipeline_options) as p:
     pcoll = (p | ReadFromText(known_args.input)
                    | beam.ParDo(WriteDecompressedFile())
-                   | beam.io.Write(bigquery.BigQuerySink(known_args.table, schema=known_args.schema)))
+                   | beam.io.Write(bigquery.BigQuerySink(known_args.table, schema=known_args.schema, validate = True)))
 
 def truncate_content(s):
     dic = json.loads(s) 
     dic['truncated'] = False
-    if sys.getsizeof(s) > THERESHOLD:
+    dic['no_records'] = 1
```

Comment:
Lets call this "records_count", the "no_" prefix can get confusing. :)

Reply:
Done.